netbeans/README.txt

```diff
@@ -0,0 +1,14 @@
+Steps to build and release the NetBeans Checker Framework plugin:
+1. Copy Checker Framework artifacts to the netbeans/CheckerFrameworkPlugin/release directory
```

Comment:
Which artifacts?  Just checker.jar or all the jar files in $CHECKERFRAMEWORK/checker/dist?